i3lock.c

```diff
@@ -811,7 +818,10 @@ int main(int argc, char *argv[]) {
                 unlock_indicator = false;
                 break;
             case 'i':
-                image_path = strdup(optarg);
+                if (optarg == NULL)
+                    image_path = "";
```

Comment:
Let's wait to see whether @stapelberg would consider merging this in the first place. I just wanted to point it out already.